python/ray/tests/kuberay/test_autoscaling_e2e.py

```diff
@@ -254,12 +262,11 @@ def testAutoscaling(self):
         )
         # 2. Trigger GPU upscaling by requesting placement of a GPU actor.
         logger.info("Scheduling an Actor with GPU demands.")
-        kubectl_exec_python_script(
-            script_name="gpu_actor_placement.py",
-            pod=head_pod,
-            container="ray-head",
-            namespace="default",
-        )
+        # Use Ray client to validate that it works against KubeRay.
+        with ray_client_port_forward(
+            head_service="raycluster-complete-head-svc", ray_namespace="gpu-test"
```

Comment:
What's the port it's forwarding? Should it be part of arguments?

Reply:
never mind, I find that client_port 10001 is the default arg in `ray_client_port_forward` definition.

Reply:
10001 is the default port for Ray client, so it seemed to make sense as a default set for this function.